user/helm-chart/README.md

```diff
@@ -0,0 +1,58 @@
+# Deploy **KubeStellar** service in a cluster using Helm
+
+Table of contests:
+- [Deploy **KubeStellar** service in a cluster using Helm](#deploy-kubestellar-service-in-a-cluster-using-helm)
+  - [Deploy **KubeStellar** in a **Kubernetes** cluster (**Kind** cluster)](#deploy-kubestellar-in-a-kubernetes-cluster-kind-cluster)
+  - [Deploy **KubeStellar** in an **OpenShift** cluster](#deploy-kubestellar-in-an-openshift-cluster)
+
+## Deploy **KubeStellar** in a **Kubernetes** cluster (**Kind** cluster)
+
+[Create a **Kind** cluster with the `extraPortMappings` for port `1024` and an **nginx** ingreass with SSL passthrough.](../yaml/README.md)
+
+Deploy **KubeStellar** `stable` in a `kubestellar` namespace:
+
+```shell
+helm install kubestellar . --values values-kubernets.yaml
+```
+
+In particular the [values-kubernets.yaml](./values-kubernets.yaml) sets:
+
+```yaml
+clusterType: Kubernetes # OpenShift or Kubernetes
+HOSTNAME: "kubestellar.svc.cluster.local" # leave it empty to let the cluster set it up

```

Comment:
What does this variable even define?  Is it something that a user cares about?

Reply:
I removed these in accordance with the changes in the previous comment